apps/sel4test-tests/src/tests/faults.c

```diff
@@ -1080,7 +1080,7 @@ static int test_vm_enter_non_vm(env_t env)
     err = api_tcb_set_space(get_helper_tcb(&helper), fault_ep, env->cspace_root,
                             api_make_guard_skip_word(guard),
                             env->page_directory, seL4_NilData);
-    test_eq(err, 0);
+    test_eq(err, (seL4_Error)seL4_NoError);
```

Comment:
The other occurrences of `seL4_NoError` in this repo are just `test_eq(err, seL4_NoError)`. The type of `seL4_NoError` is already the enum, so it looks to me like we should not need an explicit cast here. Or is there something special going on in this one?